sec_edgar_downloader/Downloader.py

```diff
@@ -52,6 +52,7 @@ def get(
         before: Optional[str] = None,
         include_amends: bool = False,
         download_details: bool = True,
+        return_meta: bool = False,
         query: str = "",
     ) -> int:
```

Comment:
Need to change the return type to be `Union[int, Tuple(int, FilingMetadata)]` to match the docstring.

Reply:
If return_meta is True, the return type is  ```Union[int, Tuple(int, FilingMetadata)]```  except it's a list of FilingMetaData objects. If that input is false, then it's an int. 

In retrospect, having different return type depending on a switch is probably a bad idea.  What if the return was a list of the meta data objects, and then the user could get the len() of the list?

Reply:
Yeah I think that might be a good solution. That would break the current API, so this would need to make it into a 5.0 release. No problem, just something to note :)

Reply:
Alternatively, I could wrap the list in an object and then the user could do something to the tune of "result.length". Sounds like you're on board with having the function return a single variable/type. I will make this change.